### PR TITLE
chore: Resolve TODOs

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: src/CSVerifier.sol [hunk 1]
```diff
@@ -283,12 +274,27 @@ contract CSVerifier is ICSVerifier {
...
+        // However, all ETH involved,
+        // including 1 or 8 ETH deposited by the attacker will remain in the Lido on Ethereum protocol
+        // Hence, the only consequence of the attack is an inconsistency in the bond accounting that can be resolved
+        // through the bond deposit approved by the corresponding DAO decision
+        //
+        // Resolution:
+        // Given no losses for the protocol,
+        // significant cost of attack (1 or 8 ETH),
+        // and lack of feasible ways to mitigate it in the smart contract's code,
+        // it is proposed to accnowleadge possibility ot the attack
```
Comment (skhomuti) [inline src/CSVerifier.sol:296]:
```suggestion
        // it is proposed to acknowledge possibility of the attack
```
i knew there is a typo in such a large comment!

**Review by skhomuti — CHANGES_REQUESTED** [review]

**Review by skhomuti — APPROVED** [review]

File: src/CSModule.sol [hunk 2]
```diff
@@ -1668,7 +1668,6 @@ contract CSModule is
         no.stuckValidatorsCount = uint32(stuckValidatorsCount);
         emit StuckSigningKeysCountChanged(nodeOperatorId, stuckValidatorsCount);
 
-        // TODO: think about reforge the condition: is `no.depositableValidatorsCount > 0` required?
```
Comment (vgorkavenko) [inline src/CSModule.sol:1671]: why?

File: src/abstract/CSBondCurve.sol [hunk 3]
```diff
@@ -157,7 +157,6 @@ abstract contract CSBondCurve is ICSBondCurve, Initializable {
         uint256[] calldata defaultBondCurvePoints
     ) internal onlyInitializing {
         uint256 addedId = _addBondCurve(defaultBondCurvePoints);
-        // TODO: Figure out how to test it
```
Comment (vgorkavenko) [inline src/abstract/CSBondCurve.sol:160]: what we decided?

File: src/CSModule.sol (hunk 2 above)
Comment (dgusakov) [reply src/CSModule.sol:1671]: This condition is needed to avoid unnecessary calls inside the block

File: src/abstract/CSBondCurve.sol (hunk 3 above)
Comment (dgusakov) [reply src/abstract/CSBondCurve.sol:160]: It is already tested. Just forgot to remove this todo.

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**